### DISCUSSION TITLE
配置成功以后报错

### DISCUSSION THREAD

**Discussion by lyizhuo** [spencerwooo/onedrive-vercel-index#344] [Q&A] [answered]
<img width="959" alt="image" src="https://user-images.githubusercontent.com/64514694/152130788-44d57f63-2f19-4a8d-b014-a0239aa0c23b.png">

**Comment by spencerwooo** [comment, marked as answer]
我看你仓库了，你最新的 deployment 显示还是之前 504 不是你这个现在的截图，我建议你根据文档从头再来一遍配置。

https://onedrive-vercel-index.spencerwoo.com/docs/getting-started

**Reply by lyizhuo** [reply]
Store tokens以后重定向到step 1，重新打开以后还是这个报错😅

**Reply by spencerwooo** [reply]
你的 OneDrive 里的 `/Public` 文件夹都有什么？

**Reply by spencerwooo** [reply]
<img width="607" alt="Screenshot 2022-02-03 at 12 00 26" src="https://user-images.githubusercontent.com/32114380/152279082-7977db79-428c-4ad8-a538-aedf8905f4b9.png">

这是你根目录返回的文件，没有 `mimeType` - 你是不是直接创建了一个空文件？
